### PR TITLE
Add cypress tests and initial documentation

### PR DESCRIPTION
**Pull request by havok2905**

### CODE REVIEW

File: TestingStrategy.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+For example, this is a simple command for retrieving a button and its use in a test.
+
+**/cypress/support/commands.ts**
+
+```typescript
+const getButton = () => {
+  return cy.get('[data-test-id="dm-screen-design-system-button"]');
+};
+
+type ChainableElement = Cypress.Chainable<JQuery<HTMLElement>>;
```
Comment (wizzah) [inline TestingStrategy.md:133]: Not a problem, just surprise. jQuery? O: neat

File: cypress/fixtures/pages/DmView/useAdventure/68c8bd92-04ff-4359-9856-8d2d6b02b69b.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
+{
+  "id": "68c8bd92-04ff-4359-9856-8d2d6b02b69b",
```
Comment (wizzah) [inline cypress/fixtures/pages/DmView/useAdventure/68c8bd92-04ff-4359-9856-8d2d6b02b69b.json:2]: What are these JSON files with hashes for names?

File: TestingStrategy.md (hunk 1 above)
Comment (havok2905) [reply TestingStrategy.md:133]: Haha, yup that was a surprise to me too. It took a bit of hunting to get that typing right.

File: cypress/fixtures/pages/DmView/useAdventure/68c8bd92-04ff-4359-9856-8d2d6b02b69b.json (hunk 2 above)
Comment (havok2905) [reply cypress/fixtures/pages/DmView/useAdventure/68c8bd92-04ff-4359-9856-8d2d6b02b69b.json:2]: Oh! If you look at the test file, you'll see code like this:

```
 cy.intercept('GET', `${TEST_API_BASE}/adventure/${TEST_ADVENTURE_ID}`, {
      statusCode: 200,
      fixture: `pages/DmView/useAdventure/${TEST_ADVENTURE_ID}`
    });
```

The string value for fixture key is just a path to this JSON file. I'm just using the id of the record as the file name, but we could use anything and call this whatever.

**Pull request merged**